rocm_getpath.cpp

```diff
@@ -25,10 +25,13 @@
 
 #include <string.h>
 #include <stdlib.h>
-#include <limits.h> /* PATH_MAX */
 #include <stdio.h>
+#if defined(_WIN32) || defined(__CYGWIN__)
```

Comment:
why a conditional statement with empty code ?  Can we invert the condition or put the condition for Linux  and put Linux code inside condition